config.json

```diff
@@ -12,7 +12,8 @@
   "require": {
     "moment": "moment",
     "_": "underscore",
-    "typogr": "typogr"
+    "typogr": "typogr",
+    "lo": "lodash"
```

Comment:
Yeah sorry... I think it got added during merge. You can remove the other
URL.
On Nov 29, 2015 11:55 PM, "Jeremy Scheff" <notifications@github.com> wrote:

> In config.json
> <https://github.com/Data4America/simple-static-stories/pull/6#discussion_r46096314>
> :
>
> > @@ -12,7 +12,8 @@
> >    "require": {
> >      "moment": "moment",
> >      "_": "underscore",
> > -    "typogr": "typogr"
> > +    "typogr": "typogr",
> > +    "lo": "lodash"
>
> Ok. Any comment on my other note?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Data4America/simple-static-stories/pull/6/files#r46096314>
> .
>